ProQuest.js

```diff
@@ -442,6 +460,10 @@ function scrape(doc, url, type, pdfUrl) {
 	});
 
 	//we may already have a link to the full length PDF
+	//or can grab it from the Download PDF link
+	if(!pdfUrl && doc.getElementById('downloadPDFLink')) {
```

Comment:
I think this should be the preferred method, if available, anyway. So no `!pdfUrl`